pkg/config/validator.go

```diff
@@ -307,7 +312,48 @@ func validateSourceBindings(sl validator.StructLevel, sources map[string]Sources
 		}
 	}
 
-	validateBindPlugins(sl, enabledPluginsViaBindings)
+	validateBoundPlugins(sl, enabledPluginsViaBindings)
+	validatePluginRBAC(sl, sources, bindings)
+}
+
```

Comment:
Please move the interface `providesPlugins` here, as this is the only usage.
Also, you can consider changing the name to `pluginProvider` or sth like this (to use nouns) 👍